README.md

```diff
@@ -46,6 +46,18 @@ test('my accessibility test', async () => {
 }
 ```
 
+You can also use axe-sarif-converter as a command line tool:
+
+```bash
+# axe-cli is used here for example purposes only; you could also run axe-core
+# using your library of choice and JSON.stringify the results.
+npx axe-cli https://accessibilityinsights.io --save ./sample-axe-results.json
+
+npx axe-sarif-converter --input-files ./sample-axe-results.json --output-file ./sample-axe-results.sarif
+```
+
+See `npx axe-sarif-converter --help` for full command line option details.
+
```

Comment:
Is it worth calling out somewhere that the way to run the CLI after building in this repo is to do `node dist/cli.js (args)`? I ended up looking at the test code to figure this out.

Reply:
Good idea, added